app/controllers/users_controller.rb

```diff
@@ -1,9 +1,13 @@
 class UsersController < ApplicationController
   def index
-    render 'index'
+    # render 'index'
+    @user = User.new
+    @user = @user.print
```

Comment:
- [ ] Kindly in Ruby on rails the syntax for getting info in a database table we use all methods on the modal. like `User.all`. and to find a specific user we use user.find(user_id). in your case you should use :point_down: 
example:

```rb
 def index
    @ = Post.all
    @user = User.find(params[:user_id])
end
```
- [ ] After this action :point_up: ,make sure the correspondent view is working .

Reply:
This issue is well resolved! ✔️